docs/upgrade-notes/rds-phase-3-data-migration-guidance.md

```diff
@@ -0,0 +1,88 @@
+
+With this bug we believe the data in Dynamo is the most reliable and Postgres is out-of-sync.
+
+### Issue 2:
+
+https://bugs.earthdata.nasa.gov/browse/CUMULUS-3024
+
+Updating an existing granule either via API or Workflow could result in datastores becoming out-of-sync if a partial granule record is provided. Our update logic operates differently in Postgres and Dynamo/Elastic. If a partial object is provided in an update payload the Postgres record will delete/nullify fields not present in the payload. Dynamo/Elastic will retain existing values and not delete/nullify.
+
+With this bug it’s possible that either Dynamo or PG could be the source of truth. It’s likely that it’s still Dynamo.
+
+### Issue 3:
+
+### https://bugs.earthdata.nasa.gov/browse/CUMULUS-3024
+
+Updating an existing granule with an empty files array in the update payload results in datastores becoming out-of-sync. If an empty array is provided, existing files in Dynamo and Elastic will be removed. Existing files in Postgres will be retained.
+
+With this bug Postgres is the source of truth. Files are retained in PG and incorrectly removed in Dynamo/Elastic.
+
+### Issue 4:
+
+https://bugs.earthdata.nasa.gov/browse/CUMULUS-3017
+
+Updating/putting a granule via framework writes that duplicates a granuleId but has a different collection results in overwrite of the DynamoDB granule but a *new* granule record for Postgres.  This *intended* post RDS transition, however should not be happening now.
+
+With this bug we believe Dynamo is the source of truth, and ‘excess’ older granules will be left in postgres.     This should be detectable with tooling/query to detect duplicate granuleIds in the granules table.
+
+### Issue 5:
+
+https://bugs.earthdata.nasa.gov/browse/CUMULUS-3024
+
+This is a sub-issue of issue 2 above - due to the way we assign a PDR name to a record, if the `pdr` field is missing from the final payload for a granule as part of a workflow message write, the final granule record will not link the PDR to the granule properly in postgres, however the dynamo record *will* have the linked PDR.       This *can* happen in situations where the granule is written prior to completion with the PDR in the payload, but then downstream only the granule object is included, particularly in multi-workflow ingest scenarios and/or bulk update situations.
+
+
+## Immediate Actions
+
+1. Re-review the issues described above
+    - GHRC was able to scope the affected granules to specific collections, which makes the recovery process much easier. This may not be an option for all DAACs.
+
+2. If you have not ingested granules or performed partial granule updates on affected Cumulus versions (questions 1 and 2 on the survey), no action is required. You may update to the latest version of Cumulus.
```

Comment:
I was going to comment above when you referenced the versions that had the fix for issue 1 that Eddie thought it would make more sense to tell them which releases were affected that they may have been doing ingest on. But since you reference the survey questions here (which listed affected versions), I think that works.

Reply:
Yeah I actually did consider more detail here around the impacted versions in the issue descriptions. I'm hesitant because I don't think these docs are really intended to introduce the issues and provide guidance for someone that hasn't been tracking the conversation. Referring back to earlier steps (like the survey) reinforces that I think.